### ISSUE TITLE
nircmd install fails

### ISSUE THREAD

**Issue by ryane** [lukesampson/scoop#3726]
```
> scoop install nircmd
Installing 'nircmd' (2.86) [64bit]
Loading nircmd-x64.zip from cache
Checking hash of nircmd-x64.zip ... ok.
Extracting nircmd-x64.zip ... done.
Linking ~\scoop\apps\nircmd\current => ~\scoop\apps\nircmd\2.86
Creating shim for 'nircmdc'.
Can't shim 'nircmdc.exe': File doesn't exist.
```

Not sure I understand why because it looks like the executable does exist if I download and extract the zip archive manually.

```
> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.18362.145
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.18362.145
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by ianharmon** [comment]
Removing `"extract_dir": ".",` from the manifest lets it install properly. It seems like a redundant line, though it was added back in August so I wonder if that's been broken since then or if the way that field is processed has changed.

I did an uncomprehensive grep on the main and extras buckets for others with this issue and only found eagleget-portable with `"extract_dir": "/",`, which also prevents it from installing.
